cmip6_downscaling/workflows/cmip6_pyramids.py

```diff
@@ -0,0 +1,169 @@
+import os
+
+import dask
+import datatree as dt
+import intake
+from adlfs import AzureBlobFileSystem
+from carbonplan_data.metadata import get_cf_global_attrs
+from carbonplan_data.utils import set_zarr_encoding
```

Comment:
Should the `get_cf_global_attrs` and `set_zarr_encoding` be included in bcsd/gard flows as well?

Reply:
yes please!

Reply:
In fact, we should specify that some of these fields are part of the intermediate and final dataset schemas